### DISCUSSION TITLE
Need a Mouseklick

### DISCUSSION THREAD

**Discussion by Mannshoch** [keepassxreboot/keepassxc#6541] [Q&A]
I have a really annoying Application that do not work with Keepass. The Problem on switching with ALT-TAB it looses the Focus on the text field. It seems also not possible to navigate by keyboard inside the text field. You have to click into it.

Is there a solution to set-up an autofill that way that it start entering text the moment I click with the mouse inside the text field?

**Comment by droidmonkey** [comment]
No there isn't and we wouldn't introduce that since we have no idea where to click

**Reply by Mannshoch** [reply]
Should be more kind of "start sequence on mouse-click"

**Reply by droidmonkey** [reply]
Just add a delay before the sequence starts: {DELAY 5000}

**Reply by Mannshoch** [reply]
For the moment I will do it that way.
I created an Issue for that.
https://github.com/keepassxreboot/keepassxc/issues/6548
